library_generation/owlbot/bin/entrypoint.sh

```diff
@@ -78,16 +79,7 @@ echo "Fixing missing license headers..."
 python3 "${scripts_root}/owlbot/src/fix-license-headers.py"
 echo "...done"
 
-# TODO: re-enable this once we resolve thrashing
```

Comment:
This was meant to restore the license header year (e.g. keep `2023` if the generated license now shows `2024`) by looking at the logs (owlbot previously worked in a git repo but we now don't do it anymore). This is just an additional cleanup